### PR TITLE
feat:aries js worker notifier to watch instead of pop

### LINKED ISSUE #1262: aries-js-worker notifier - wait for message instead of regular pop

### PR DESCRIPTION
**Pull request by sudeshrshetty**
- changed `aries.waitForNotification(topics)` to
`aries.startNotifier(topics, callback)`
- added message center in presentation html to display incoming
messages/topics in table instead of alerts
- closes #1262

Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

File: cmd/aries-js-worker/src/aries.js [hunk 1]
```diff
@@ -107,22 +125,27 @@ export const Aries = function(opts) {
             return invoke("aries", "Stop", "{}", "timeout while stopping aries")
         },
 
-        waitForNotification : async function waitForNotification(topics) {
-            return new Promise((resolve, reject) => {
-                const timer = setTimeout(_ => resolve(), 10000)
-                // subscribe for all by default if topics not provided
-                if (topics.length == 0){
-                    topics = ["all"]
+        startNotifier : function(callback, topics) {
```
Comment (troyronda) [inline cmd/aries-js-worker/src/aries.js]: Is there a TODO for stop?

File: cmd/aries-js-worker/src/aries.js [hunk 2]
```diff
@@ -53,6 +53,24 @@ async function invoke(pkg, fn, arg, msgTimeout) {
     })
 }
 
+async function waitForNotification(topics) {
+    return new Promise((resolve, reject) => {
+        const timer = setTimeout(_ => resolve(), 10000)
```
Comment (troyronda) [inline cmd/aries-js-worker/src/aries.js]: We should make 10000 a const and supply an explanation on what it is and why 10000.

Comment (sudeshrshetty) [reply cmd/aries-js-worker/src/aries.js]: I will add explanation

File: cmd/aries-js-worker/src/aries.js (hunk 1 above)
Comment (sudeshrshetty) [reply cmd/aries-js-worker/src/aries.js]: Yes, will add a TODO and will add stop functionality

File: cmd/aries-js-worker/index.html [hunk 3]
```diff
@@ -42,105 +43,133 @@
...
-            // TODO whole generator logic below to be moved inside js package (aries.js)
-           async function* run() {
-                while (true)
-                    yield await aries.waitForNotification(topics)
-            }
+            aries.startNotifier(pushToMsgPanel, topics)
+        }
+
+        function pushToMsgPanel(val) {
+            console.log("incomoing msg", val.id, val.topic, val.payload)
```
Comment (troyronda) [inline cmd/aries-js-worker/index.html]: typo: incomoing

Comment (sudeshrshetty) [reply cmd/aries-js-worker/index.html]: will fix it

**Pull request merged**